homeassistant/components/fully_kiosk/strings.json

```
@@ -147,6 +147,10 @@
       "name": "Load URL",
       "description": "Loads a URL on Fully Kiosk Browser.",
       "fields": {
+        "device_id": {
+          "name": "Device ID",
+          "description": "ID of the device."
```

The description should differ a bit more from the name ;-)

```suggestion
          "description": "The target device for this action."
```